CHANGELOG.md

```
@@ -3,6 +3,150 @@
+ - There are no breaking changes in this release.
+
+Packages with other changes:
+
+ - [`flutter_stripe_web` - `v7.4.0`](#flutter_stripe_web---v740)
+
+---
+
+#### `flutter_stripe_web` - `v7.4.0`
+
+
+## 2026-03-09
```

markdownlint-cli2 (0.21.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)